package.json

```diff
@@ -95,17 +103,17 @@
     "sass-loader": "^6.0.7",
     "style-loader": "^0.13.2",
     "url-loader": "^0.5.7",
-    "webpack": "^1.15.0",
+    "webpack": "^3.8.1",
     "yargs": "^4.3.1"
   },
   "scripts": {
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "clean": "rimraf dist && rimraf coverage*",
-    "build": "npm run clean && webpack --progress --colors --mode=production --target=web",
+    "build": " npm run clean && NODE_ENV=\"production\" webpack --progress --colors",
     "build:dev": "npm run clean && webpack --progress --colors --mode=dev --target=web",
     "build:prod": "npm run test && npm run build",
     "build:deploy": "webpack --progress --colors --mode=deploy --target=web",
-    "watch": "webpack --progress --colors --watch --mode=deploy --target=web",
+    "watch": "NODE_ENV=\"development\" webpack --progress --colors --watch --cache=false",
```

Comment:
@IAMOTZ so the previous webpack version had the `--mode` option and `deploy` was just a value to pass around and use in the config to know what mode you were actually running ie. `production`, `dev` etc, version 4 also has the mode option but 3 doesn't.

Reply:
it works as it should normally

Reply:
I should prolly update the other script commands also seeing as they have `deploy` on them

Reply:
Well, I just tried it out and it is not working. This is a snapshot of the error:

![screen shot 2018-08-07 at 11 00 50 am](https://user-images.githubusercontent.com/26048536/43769324-608e5538-9a31-11e8-8592-96a42b20726c.png)
Since you updated this part of the configuration:
```
else if (env === 'deploy') {
	outputFile = `${outputFile}.js`;
	vendorOutputFile = "vendor.bundle.js";
	outputPath = `${config.LOCAL_OWA_FOLDER}${config.LOCAL_OWA_FOLDER.slice(-1) != '/' ? '/' : ''}${THIS_APP_ID}`;
	devtool = 'source-map';

}
```
to this: 
```
if (env === 'development') {
	outputFile = `${outputFile}.js`;
	vendorOutputFile = "vendor.bundle.js";
	outputPath = `${config.LOCAL_OWA_FOLDER}${config.LOCAL_OWA_FOLDER.slice(-1) != '/' ? '/' : ''}${THIS_APP_ID}`;
	devtool = 'eval-source-map';
}
```
The `build:deploy` script is supposed to build the project and inject it in the OpenMRS standalone but with the changes you are making in the `webpack.config.js`, the `build:deploy` script needs to be also updated to something like this:
```
"build:deploy": "NODE_ENV=\"development\" webpack --progress --colors --target=web",
```

Reply:
good catch mahn! updating scripts asap